src/providers/ldap/sdap_async_connection.c

```diff
@@ -337,6 +338,12 @@ static void sdap_sys_connect_done(struct tevent_req *subreq)
         goto fail;
     }
 
+    stat_info = talloc_asprintf(state, "server: [%s] START TLS",
+                                sdap_get_server_ip_str(state->sh));
+    if (stat_info == NULL) {
+        DEBUG(SSSDBG_OP_FAILURE, "Failed to create info string, ignored.\n");
+    }
+
```

Comment:
I guess `stat_info` is supposed to be used as an arg of following `sdap_op_add()`?

Reply:
And why does message say "START TLS"?

Reply:
fixed, and is says `START TLS` because it tracks the Start TLS LDAP extended operation.